tests/test_suggester.py

```diff
@@ -276,6 +300,29 @@ def test_suggest_symlink_configuration():
     )
 
 
+def test_deprecated_noop_suggestion():
+    config = """{"links": {
+"2050.02-py58": "2050.02.00-py58",
+"2050.01-py58": "2050.01.00-py58",
+"deprecated-py58": "2050.01-py58",
+"stable-py58": "2050.02-py58"
+}}"""
+    repo = _mock_repo(config)
+
+    args = Namespace(
+        git_ref="master",
+        release="2050.02.00-py58",
+        mode="stable",
+        symlink_conf_path="foo.json",
+        joburl="http://job",
+        jobname="job",
+    )
+
+    repo.create_pull.assert_not_called()
+
+    assert suggest_symlink_configuration(args, repo) is None
```

Comment:
Not really sure if I understand this test, ie. shouldn't we test actually the content of the suggestion too?